components/IscnRegisterForm.vue

```diff
@@ -89,6 +94,30 @@
       >
         <ContentFingerprintLink v-if="ipfsHash"  :item="formattedIpfs" />
         <ContentFingerprintLink v-if="uploadArweaveId" :item="formattedArweave" />
+        <ContentFingerprintLink v-for="f, i in customContentFingerprints" :key="f + i" :item="f" />
+        <div
+          v-if="!fileData"
+          :class="[
+            'flex',
+            'mt-[12px]',
+            'gap-[8px]'
+          ]"
+        >
+          <TextField
+            v-model="contentFingerprintInput"
+            class="w-full"
```

Comment:
Placeholder?

Reply:
Actually no idea what would be a suitable placeholder...